nix/tullia.nix

```diff
@@ -0,0 +1,78 @@
+self: system:
+
+let
+  ciInputName = "GitHub event";
+in rec {
+  tasks = let
+    inherit (self.inputs.tullia) flakeOutputTasks taskSequence;
+
+    common = {
+      config,
+      ...
+    }: {
+      preset = {
+        # needed on top-level task to set runtime options
+        nix.enable = true;
+
+        github-ci = {
+          enable = config.actionRun.facts != {};
+          repo = "input-output-hk/cardano-node";
+          sha = config.preset.github-ci.lib.getRevision ciInputName null;
+        };
+      };
+    };
+
+    mkHydraJobTask = flakeOutputTask: {...}: {
+      imports = [common flakeOutputTask];
+
+      # some hydra jobs run NixOS tests
+      env.NIX_CONFIG = ''
+        extra-system-features = kvm
+      '';
+
+      memory = 1024 * 8;
+      nomad.resources.cpu = 10000;
+    };
+    mkHydraJobTasks = __mapAttrs (_: mkHydraJobTask);
+
+    hydraJobTasks   = mkHydraJobTasks (flakeOutputTasks ["hydraJobs"] self);
+    # TODO this should this be using `hydraJobsPr` instead of `hydraJobs`?
+    # Where does that come from?
+    hydraJobPrTasks = mkHydraJobTasks (flakeOutputTasks ["hydraJobs"] self);
```

Comment:
Custom output of cardano-node I guess. Having only `.#hydraJobs` is fine.
```suggestion
```

Reply:
switched to `ciJobs` that has system first